websites/D/Doodstreams/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "638080361179512853",
+		"name": "dark_ville"
+	},
+	"service": "Doodstreams",
+	"description": {
+		"en": "Doodstream redefines video sharing with a commitment to unrivaled HLS streaming, unlimited bandwidth and seamless user experience."
+	},
+	"url": "doods.pro",
+	"regExp": "(ww(w)?([0-9])?[.])?dood(s)?[.][a-z]*",
```

Comment:
this regex is probs too narrow as their domain changes a lot cause they had doodstream.com(.tr) too